app/components/base_ui/ImageLinkWrapper/index.jsx

```diff
@@ -9,34 +9,56 @@ const ImageLinkWrapper = ({
   disabled,
   image,
   isSquare,
+  isCircle,
   onClick,
   route,
   size,
   ...restProps
-}) => (
-  <StyledLink disabled={disabled} onClick={onClick} to={route}>
-    <StyledImage
-      alt={alt}
-      isSquare={isSquare}
-      size={size}
-      src={image}
-      {...restProps}
-    />
-  </StyledLink>
-);
+}) => {
+  const onLoadHandler = React.useCallback(
```

Comment:
Let's import `useCallback` above --> `import React, { useCallback } from 'react';` above, and then we can just do the following:
```
const onLoadHandler = useCallback(

```